### ISSUE TITLE
Can I get  crash report  when client's APP is crash ?????

### ISSUE THREAD

**Issue by oliverwithdason** [kstenerud/KSCrash#211]
Can I get  crash report  when client's APP is crash? if can please let me know, thank you a lot!
I have attempted three method to send reports, but it seems to no work.
1. when the crash occurs, I try to catch the exception(in KSCrashSentry_NSException), I use CFNetwork to POST information,But I can't send report.in the other world.it isnt best way to accomplish it ;
2. When the APP will terminating. I try to send report,but it doesnt work too;
3. When report finish to write. I try to send report use CFNetwork.but it not work too.
Can you give me lighted way to send the report out ? It wired me out! thank you a lot! Wait you reply!thank you a lot agin agin!

**Comment by oliverwithdason** [comment]
Who can help me???????????????

**Comment by kstenerud** [comment]
You can't send a crash report anywhere from a crashed context. Once the app is crashed, all you can do is write a file to disk and then wait for the user to relaunch the app before attempting to send the report. This is what KSCrash does already.

**Comment by naftaly** [comment]
Yes, crash reports are saved to disk and can be retrieved and sent when the app restarts.
